crates/modelardb_server/src/optimizer/model_simple_aggregates.rs

```diff
@@ -627,27 +757,31 @@ impl Accumulator for ModelSumAccumulator {
         Ok(())
     }
 
+    /// Panics as this method should never be called.
     fn merge_batch(&mut self, _states: &[ArrayRef]) -> Result<()> {
         unreachable!()
     }
 
+    /// Return the current state of the [`Accumulator`].
     fn state(&self) -> Result<Vec<ScalarValue>> {
+        // Match SumAccumulator in repository/datafusion/physical-expr/src/aggregate/sum.rs.
         Ok(vec![
             ScalarValue::Float32(Some(self.sum)),
             ScalarValue::from(self.count),
         ])
     }
 
+    /// Panics as this method should never be called.
     fn evaluate(&self) -> Result<ScalarValue> {
         unreachable!()
     }
 
+    /// Return the size of the [`Accumulator`] including `Self`.
     fn size(&self) -> usize {
         mem::size_of_val(self)
     }
 }
 
-//Avg
```

Comment:
It seems that this struct is missing a doc comment. Something similar to the one for `Count` should suffice.

Reply:
Fixed.